components/nexus-bootstrap/src/main/java/org/sonatype/nexus/bootstrap/jetty/InstrumentedSelectChannelConnector.java

```diff
@@ -1,99 +0,0 @@
-/*
```

Comment:
Both `InstrumentedSelectChannelConnector ` and `InstrumentedSslSelectChannelConnector` are referenced in `etc/jetty.xml` and `etc/jetty-https.xml` - so those files will need updating (to presumably use the new `InstrumentedConnectionFactory`?).